webpack.common.config.js

```diff
@@ -25,6 +25,7 @@ var entry = {
     // JS
     'base-page': staticPath('js/pages/base-page.js'),
     'home-page': staticPath('js/pages/home-page.js'),
+    'landing-page': staticPath('js/pages/landing-page.js'),
```

Comment:
No, I didn't add the home route in my PR, I branched off of Caner's dashboard PR and the /home route was already there with Placeholder text.  I just placed the quick search project stuff on that home page.  Multiple people have been developing widgets for the /home page, I'm doing the Quick Search Projects and New and Noteworthy, and Henrique is doing an activity widget.  

Issue https://openscience.atlassian.net/browse/OSF-5605 will be resolved first because it is a blocker to releasing Phase 0 of the dashboard into production (Caner's PR is currently being reviewed: https://github.com/CenterForOpenScience/osf.io/pull/4887/files).  Quick Search projects is part of Phase 1.